changelog/565.doc.rst

```diff
@@ -0,0 +1 @@
+Update sphinx from 4.5.0 to version 5.1.0, and take advantage of new options.
```

Comment:
```suggestion
Update sphinx from 4.5.0 to version 5.1.1, and take advantage of new options.
```
just `5.1` is fine too.

Reply:
https://github.com/DisnakeDev/disnake/pull/565/commits/2e8241260b8896ece657b7cd5a45886808179d0a